tests/ui/lint/implicit_autorefs.stderr

```diff
@@ -2,9 +2,16 @@ warning: implicit autoref creates a reference to the dereference of a raw pointe
   --> $DIR/implicit_autorefs.rs:10:13
    |
 LL |     let _ = (*ptr)[..16];
-   |             ^^^^^^^^^^^^
+   |             ^^---^^^^^^^
+   |               |
+   |               raw pointer coming from here: `*const [u8]`
```

Comment:
I have a hard time interpreting this error. It seems to say that the raw pointer comes from `*const [u8]` but that makes no sense?

Reply:
Yeah, it's not great. What I want to say it is the raw pointer (whose type is X) is coming from the pointed place. Maybe I should just use that formulation?

> raw pointer of type `*const [u8]` is coming from here

Reply:
Or maybe it should be more generic and just say:

> has type `*const [u8]`

Reply:
Or maybe "this raw pointer has type T"? Or "raw pointer produced here has type T"?

Reply:
Sure, changed.